core/examples/http_bench.rs

```diff
@@ -181,7 +156,11 @@ fn main() {
     });
 
     let mut isolate = deno::Isolate::new(startup_data, false);
-    isolate.set_dispatch(dispatch);
+    isolate.register_op("listen", serialize_http_op(op_listen));
+    isolate.register_op("accept", serialize_http_op(op_accept));
+    isolate.register_op("read", serialize_http_op(op_read));
+    isolate.register_op("write", serialize_http_op(op_write));
+    isolate.register_op("close", serialize_http_op(op_close));
```

Comment:
I see, op_ids are assigned here... that answers [my previous question](https://github.com/denoland/deno/pull/3002/files#r328159510).